week2.6.3.count_words.py

```diff
@@ -0,0 +1,37 @@
+# region count words - this 2
+
+def count_words(text):
+    """
+    Count the number of words in a text and their lengths.
+
+    Args: text (str): The text to count words from.
+
+    Returns:  dict: A dictionary where keys are words and values are their lengths.
+    """
+    # Clean the text

```

Comment:
I would use documentation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/